### PR TITLE
Fixed AngularSelect

### PR DESCRIPTION
**Pull request by psav**
{{pytest: -k test_provider_crud}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit c7f9b331496407785bd1d7eaa30dc4faf6542420:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a0e4d962f5b6e265c672ac0df3201093e5786bf2:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by psav** [comment]
@seandst @mfalesni See any issues with this approach? without it selects just don't function properly.

**Comment by dajoRH** [comment]
Lint report for commit 2b053bf3b955742329bb318c4b38d18235e21816:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 82d6e84c6a27ef0d7aa12fd7b23d2a8843905ba6:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/cloud/provider.py [hunk 1]
```diff
@@ -101,6 +101,9 @@ class Provider(Updateable, Pretty, BaseProvider):
     add_provider_button = deferred_verpick(
         {version.LOWEST: form_buttons.FormButton("Add this Cloud Provider"),
          '5.5': form_buttons.FormButton("Add")})
+    save_button = deferred_verpick(
+        {version.LOWEST: form_buttons.FormButton("Save Changes"),
+         '5.5': form_buttons.FormButton("Save changes")})
```
Comment (mfalesni) [inline cfme/cloud/provider.py]: Y U NO HAVE CASE INSENSITIVITY XPATH ....

Comment (seandst) [inline cfme/cloud/provider.py]: https://github.com/RedHatQE/cfme_tests/blame/master/cfme/web_ui/form_buttons.py

:D

**Pull request merged**